scripts/android/helper.js

```diff
@@ -32,20 +32,47 @@ function addDependencies(buildGradle) {
 }
 
 /*
- * Add 'google()' to the repository repo list
+ * Add 'google()' and Crashlytics to the repository repo list
  */
 function addRepos(buildGradle) {
   // find the known line to match
   var match = buildGradle.match(/^(\s*)jcenter\(\)/m);
```

Comment:
Perhaps to make this code easier to read and maintain, we should add two specific comments to the gradle file that we look for instead of looking for the location of two `jcenter()` entries.  Just a suggestion.

Reply:
Testing this out, it appears it might not be possible since I can't figure out how to get a comment to be added in the gradle file generated during the build.